qemu/tests/hotplug_mem_stress.py

```diff
@@ -0,0 +1,49 @@
+from virttest import utils_test
+from virttest import error_context
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu memory hotplug test:
+    1) Boot guest with -m option
+    2) Add movable_node to guest kernel line
```

Comment:
Could you explain what and where is this ?

Reply:
It's used to unplug, the code is in L23.

Reply:
Thanks!